spot-contracts/contracts/vaults/RolloverVault.sol

```diff
@@ -74,6 +92,10 @@ contract RolloverVault is
     /// @dev The maximum number of deployed assets that can be held in this vault at any given time.
     uint256 public constant MAX_DEPLOYED_COUNT = 47;
 
+    /// @dev The maximum percentage fee paid in the underlying assets by users who "meld".
+    ///      The meld fee paid by the user in the underlying asset.
+    uint256 public constant MAX_MELD_FEE_PERC = 5 * UNIT_PERC; // (5%)
```

Comment:
The right value, and whether it should be configurable through governance, is worthy of some discussion.

I'm feeling like this is low, since practically speaking, in the final week with the current config and the linear dropoff, the applied fees will be very small.

Reply:
I'm leaning toward making the vault completely immutable and thus having no governance leavers at all. 

Also, there's no correct answer for what the "right" fee is. We generally feel something in the order of magnitude  of 5% for the duration makes sense. With the linear drop-off, it should be less than 1.25% in the last week. Seems like a reasonable trade-off for market actors to either hold to maturity or redeem early and pay the fee.

Reply:
I'm a little nervous that we don't really know what "reasonable" is yet. I think I'm still leaning towards governance-configurable fee in the beginning, and we can always lock it down later.